schema.sql

```diff
@@ -11,3 +11,19 @@ CREATE TABLE species(id SERIAL PRIMARY KEY, name VARCHAR(255));
 ALTER TABLE animals DROP COLUMN species;
 ALTER TABLE animals ADD COLUMN species_id INTEGER REFERENCES species(id);
 ALTER TABLE animals ADD COLUMN owner_id INTEGER REFERENCES owners(id);
+
+CREATE TABLE vets(id SERIAL PRIMARY KEY, name VARCHAR(255), age INTEGER, date_of_graduation DATE);
+
+CREATE TABLE visits(animal_id INTEGER,
+  vet_id INTEGER, 
+  PRIMARY KEY(animal_id, vet_id),
+  visit_date Date,
+  FOREIGN KEY (animal_id) REFERENCES animals (id)
+  FOREIGN KEY (vet_id) REFERENCES vets (id));
```

Comment:
- [ ] [Optional]👏 Great effort! However, I noticed a syntax error in your SQL `CREATE TABLE` operation. To resolve this issue, kindly review the SQL statement responsible for creating the table and make necessary adjustments. This will ensure that your database structure is accurately defined. Keep up the good work! 👍💼